features/machine/machine.feature

```diff
@@ -487,30 +487,24 @@ Feature: Machine features testing
   @admin
   @4.10 @4.9
   Scenario: Node labels and Affinity definition in PV should match
-    Given I have a project
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    And I use the "openshift-machine-api" project
```

Comment:
Prefer creating test resources in newly created test project, rather than use existing system projects.

Reply:
If I use "Given I have a project " instead of using the openshift-machine-api project , I get [error](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/347065/console) 

`03-09 15:22:21.365        [09:52:19] INFO> Shell Commands: oc auth can-i delete project -q --kubeconfig=/home/jenkins/ws/workspace/ocp-common/Runner/workdir/ocp4_admin.kubeconfig -n openshift-machine-api
03-09 15:22:21.365        
03-09 15:22:21.365        STDERR:
03-09 15:22:21.365        Warning: resource 'projects' is not namespace scoped in group 'project.openshift.io'`

Hence avoiding it . It seems safe to use the openshift-machine-api project in this [case](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/347077/console) . Please suggest .

Reply:
Please consider the comments together.

Reply:
Thanks @liangxia  , getting errors while using them , have shared on slack , have a look when time permits ..